understanding/22/focus-not-obscured-minimum.html

```diff
@@ -56,12 +56,21 @@ <h4>Modal dialogs</h4>
+
+         <ul>
+            <li><strong>When the additional content appears, it displaces existing content.</strong> An accordion is an example of this. When an accordion is opened, the disclosed content shifts existing content further down the page. Since the new content does not obscure existing content, it cannot obscure the item with focus.</li>
+            <li><strong>When the additional content appears, existing content reflows.</strong> The popout sidebar on the <a href="https://www.w3.org/TR/WCAG22/">WCAG standard</a> is an example of this pattern. When the side menu is activated, it discloses a new section of information along the left side of the page. The main content area is reduced horizontally to accommodate this, and the existing main content reflows to fit in the thinner space. As a result, there is no overlapping content between the two sections; the item receiving focus, whether in the left navigation or in the main content, will not be obscured by the other section.</li>
+            <li><strong>When the additional content is disclosed, it takes focus and the tab ring is constrained to the new content until it is dismissed.</strong> This modality is somewhat like a dialog, in that a user cannot navigate beyond the disclosed content by keyboard without dismissing it first (typically by pressing Esc). However, unlike in a modal dialog, in some implementations a pointer user may be able to interact with content outside the disclosed section without dismissing it. Since this pattern potentially creates an inequitable experience between keyboard and pointer users, it should be used cautiously. That said, it does prevent the disclosed content from obscuring the keyboard focus in the main content, and thus should pass this SC. </li>
+            <li><strong>The disclosure expands into an area of the page containing no other content.</strong> Many pages are designed with wide margins, providing significant white space into which new content can be disclosed. Many chatbots and toasters are designed to 'slide up' into the right unpopulated side of a page. Where authors are careful to ensure content is not obscured at each breakpoint in a response design, no obscuring of other operable content need occur.</li>
+            <li><strong>When focus leaves the additional content, it is hidden or collapsed.</strong> This is very similar to patterns discussed next under Non-persistent disclosed information. A distinguishing factor can be that the user's last point of interaction in the disclosure is preserved (it persists) even though it may be hidden until a user returns. Some trees and left navigation pattterns behave this way. </li>
+         </ul>
+         <p>Where disclosed content persists and causes existing content to be obscured, it will be at risk of failing this criterion if it prevents users from seeing the focus indicator.</p>
```

Comment:
what about disclosures that persist/overlay other content, and allow keyboard focus to leave the disclosure, but allow click and tap outside of the disclosure content, or esc key to dismiss the disclosure content at any point that it is open?   

the persistent disclosure that traps focus but allows outside clicks is mentioned, but not this variant where outside clicks/taps or esc key would dismiss the overlaying disclosure.

Reply:
@scottaohara , I'd like to explore wording that mirrors some of the behaviour required in On Hover or Focus to try to allow for such an interaction mechanism. Something like:

> Where content disclosed by the user can obscure other content, if the user can reveal the item with focus without moving keyboard focus (for example, by pressing Esc), such disclosed content is not considered to fully obscure the item receiving focus.

Reply:
i think that's on the right track.  Thank you!  That'd definitely alleviate my concerns if that sort of text was added